webdriver/chrome.go

```diff
@@ -39,10 +39,10 @@ func ChromeWebDriver() (*selenium.Service, selenium.WebDriver, error) {
 		options = append(options, selenium.StartFrameBuffer())
 	}
 	// Specify the path to ChromeDriver in order to use Chrome.
+	// FIXME: chromeDriverPath seems to be ignored?
 	// Output debug information to STDERR.
 	options = append(options, selenium.Output(os.Stderr))
 
-	selenium.SetDebug(true)
```

Comment:
Could we plumb a flag into here in case we want to debug webdriver tests? I'd be fine with commenting out the code an adding a `TODO`, since this PR's already so large.

Reply:
Added TODO (and in `firefox.go` as well).